sdk/identity/identity/src/credentials/visualStudioCodeCredential.ts

```diff
@@ -145,18 +129,69 @@ export class VisualStudioCodeCredential implements TokenCredential {
     checkUnsupportedTenant(this.tenantId);
   }
 
+  private async loadBrokerPlugin() {
+    try {
+      // @ts-ignore
+      const { nativeBrokerPlugin } = await import("@azure/identity-broker");
+      return nativeBrokerPlugin;
+    } catch (error) {
+      // Package not installed, return null or fallback
```

Comment:
It might make sense to check the kind of error and rethrowing if its not the one we expect. I don't remember which error we expect but I think it's  one with field `code: 'ERR_MODULE_NOT_FOUND'`